### ISSUE TITLE
Move tutorial pager controls up slightly

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#150]
Helping debug an Appium issue where it cannot find the back and next accessibility elements. They may be out of the view port and not accessible so I'm going to move them up slightly in hopes this improves things.
